### DISCUSSION TITLE
pyTelegramBotApi handlers and AWS Lambda

### DISCUSSION THREAD

**Discussion by sphinks** [eternnoir/pyTelegramBotAPI#2013] [General]
I wonder, how I could utilise pyTelegramBotApi handler decorators with AWS Lambda code (webhook)? In every example I could find the idea is to do manual parsing of message and detect the key word. Decorators not used. However, I suppose to make it possible to use decorators I need to pass body of event to some general handling method that next will route that to proper handler. Is it possible?

**Comment by Badiboy** [comment]
https://github.com/eternnoir/pyTelegramBotAPI/blob/master/examples/serverless/aws_lambda_function.py

**Reply by sphinks** [reply]
Great! That helps a lot! Thank you!

**Reply by nahomderese** [reply]
This helped me a lot, Thanks mate !
